AUTHORS.rst

```diff
@@ -0,0 +1,13 @@
+=======
+Credits
+=======
+
+Development Lead
+----------------
+
+* Lukas Heumos <lukas.heumos@posteo.net>
+
+Contributors
+------------
+
+None yet. Why not be the first?
```

Comment:
I'd just suggest a list of contributors. No need to identify a Dev Lead.

Reply:
Absolutely. Sorry, this is autogenerated. Will fix